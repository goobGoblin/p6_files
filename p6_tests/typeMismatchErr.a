typeMismatchError : () -> int {
  a : int;
  b : bool;
  
  a = 5;
  b = true;
  
  a = b;
  
  return a;
}
