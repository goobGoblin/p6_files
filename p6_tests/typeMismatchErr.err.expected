FATAL [8,3]-[8,8]: Invalid assignment operation
